openslides/core/static/js/core/projector.js

```diff
@@ -140,11 +156,48 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
 
 .controller('SlideSpeakerOverlayCtrl', [
     '$scope',
-    function($scope) {
+    'Motion',
+    'Assignment',
+    'Agenda',
+    function($scope, Motion, Assignment, Agenda) {
         // Attention! Each object that is used here has to be dealt on server side.
         // Add it to the coresponding get_requirements method of the ProjectorElement
         // class.
-	$scope.visible = $scope.element.visible;
+        Motion.bindAll({}, $scope, 'motions');
+        Assignment.bindAll({}, $scope, 'assignments');
+        $scope.visible = $scope.element.visible;
+        
+        //get list of speakers
+        var displayeditem = null;
+        angular.forEach($scope.elements, function(element) {
+            if (element.name == "motions/motion") {
+                var currentmotion = Motion.find(element.id);
+                //TODO displayeditem = currentmotion.agenda_item;
+            } else if (element.name == "core/customslide") {
+                displayeditem = element.id;
+            } else if (element.name == "assignments/assignment") {
+                
+                var currentassign = Assignment.find(element.id);
+                
+                //TODO I want to get the 'value.agenda_item_id' from this currentassign object.
+                
```

Comment:
@emanuelschuetze:

Here I am in the projector/speakeroverlay element. I want to connect to the currently displayed agenda item, to get their list of speakers. `currentassign` is the currently displayed assignment item, which holds in itself the agenda information.

However, I fail miserably at accessing its' `agenda_item_id` value (which *is* existant and accessible for the user in question, as the values are sent somewhere nested in the answer - I can see them in Inspektor). Below are some of the accesses I tried. I may be missing something really simple.